src/d3a/d3a_core/simulation.py

```diff
@@ -94,7 +96,12 @@ def __init__(self, setup_module_name: str, simulation_config: SimulationConfig =
 
         self._load_setup_module()
         self._init(**self.initial_params)
+        # TODO: Deprecated event mechanism since D3ASIM-1328 is about to be merged.
+        # Candidate for deletion.
```

Comment:
Dont we need that when running locally setup files that contain events?

Reply:
No, this is from the event mechanism that Brainbot had introduced before taking over the project. Unfortunately it was partially implemented, that's why we introduced the new event mechanism. As far as I understand, the old event mechanism is not used at all anymore (apart from, maybe, unit tests) and we can safely remove it.

Reply:
Ok, understood. So please go ahead and delete it. Thanks a lot!